deeplearning4j/deeplearning4j-nlp-parent/deeplearning4j-nlp/src/main/java/org/deeplearning4j/models/embeddings/learning/impl/elements/CBOW.java

```diff
@@ -212,38 +222,81 @@ public double doExec(List<BatchItem<T>> items, INDArray inferenceVector) {
                         maxWinWordsCols = curr;
                 }
 
-                INDArray inputWindowWords = Nd4j.createUninitializedDetached(DataType.INT32, items.size(), maxWinWordsCols);
+            /*    INDArray inputWindowWords = Nd4j.createUninitializedDetached(DataType.INT32, items.size(), maxWinWordsCols);
                 INDArray inputWordsStatuses = Nd4j.createUninitializedDetached(DataType.INT32, items.size(), maxWinWordsCols);
                 INDArray randoms = Nd4j.createUninitializedDetached(DataType.INT64, items.size());
                 INDArray alphas = Nd4j.createUninitializedDetached(DataType.DOUBLE, items.size());
                 INDArray currentWindowIndexes = Nd4j.createUninitializedDetached(DataType.INT32, items.size());
                 INDArray codes = Nd4j.createUninitializedDetached(DataType.INT8, items.size(), maxCols);
                 INDArray indices = Nd4j.createUninitializedDetached(DataType.INT32, items.size(), maxCols);
                 INDArray numLabelsArray = Nd4j.createUninitializedDetached(DataType.INT32, items.size());
```

Comment:
please don't leave code commented out like that.

Reply:
Done. Sorry missed that.